### PR TITLE
tentative working version of better chain event emails

### PR DESCRIPTION
**Pull request by dillchen**
The current immediate email notification for chain events is not descriptive. I've augmented `createNotificationEmailObject`  in a few ways.

1. It accepts both `IPostNotificationData` and `IChainEventNotificationData`
2. It uses `@commonwealth/chain-events` to pull labels for chain events.

### CODE REVIEW

File: server/scripts/emails.ts [hunk 1]
```diff
@@ -49,8 +76,8 @@ export const createNotificationEmailObject = (notification_data: IPostNotificati
     dynamic_template_data: {
       notification: {
         subject: subjectLine,
-        title: decodedTitle,
-        path,
+        title: label ? decodedTitle : label.label,
+        path: label ? path: label.linkUrl,
```
Comment (raykyri) [inline server/scripts/emails.ts:80]: This looks reversed to me?

Comment (zakhap) [reply server/scripts/emails.ts]: Fixed this!

**Comment by zakhap** [comment]
@dillchen Added a `body` value to the payload that goes to SendGrid, which should be used instead of redundantly using the subject line before the redirect link back to CW.

**Comment by zakhap** [comment]
Links in emails generated by the SubstrateEvent's labeler, to test/verify: 
- [ ] Validator Events
  - [x] Bonded
  - [x] Rewarded
- [ ] Council Events
- [ ] Democracy Events
- [ ] Preimage Events
- [ ] Vote Events

**Comment by raykyri** [comment]
What's the latest status on this?

**Comment by zakhap** [comment]
> What's the latest status on this?

Generally, email notification enrichment should be working. I've not tested all the different Chain Event possibilities except for the 2 within Validator Events I've checked off above.

**Review by raykyri — APPROVED** [review]

**Pull request merged**